content/applications/hr/appraisals/skills_evolution.rst

```diff
@@ -98,6 +98,54 @@ appear. In this example, only :guilabel:`Marc Demo` meets these criteria.
    :align: center
    :alt: The employees with expert Javascript skills.
 
+Use case: Assess highest improvement
+====================================
+
+Another way to modify the :guilabel:`Appraisal Skills Report` is to identify the employee who has
+the highest amount of improved skills over a specific period of time. In this example, the employee
+with the most amount of improved skills for the third quarter is being determined.
+
+To view this information, first remove the default filter in the search bar. Next, click the
+:icon:`fa-caret-down` :guilabel:`(down arrow)` icon in the search bar, then click
+:guilabel:`Improvement` beneath the :icon:`fa-filter` :guilabel:`Filters` column. Enabling this
```

Comment:
oooof i see an option for "regression" :skull:

Reply:
Yeah.. I tried to focus on the positive and not use that as the example. i figured people will find it if they look =D